### PR TITLE
re-export original from immer

### LINKED ISSUE #829: compare original state with reference equality
I have a reducer created using `createSlice`. Inside one of the case reducer, I'd like to compare with the original state with reference equality. How do I achieve this?

```
const slice = createSlice({
  name: 'filter',
  initialState: emptyConstraint,
  reducers: {
    addPlain(state, action: PayloadAction<{ parent: Constraint.AsObject }>) {
      if (state === action.payload.parent) { // <<<< how should I do equality check? do I need to import: https://immerjs.github.io/immer/docs/original
          // do something with state
      }
    },
})
```

### PR DESCRIPTION
**Pull request by jackielii**
fixes #829

### CODE REVIEW

File: docs/api/otherExports.mdx [hunk 1]
```diff
@@ -30,6 +30,10 @@ The default immutable update function from the [`immer` library](https://immerjs
 
 [The `current` function](https://immerjs.github.io/immer/docs/current) from the [`immer` library](https://immerjs.github.io/immer/), which takes a snapshot of the current state of a draft and finalizes it (but without freezing). Current is a great utility to print the current state during debugging, and the output of `current` can also be safely leaked outside the producer.
 
+### `original`
+
+[The `original` function](https://immerjs.github.io/immer/docs/original) from the [`immer` library](https://immerjs.github.io/immer/), which returns the original object. This is particularly useful for referencial equality check in reducers.
```
Comment (markerikson) [inline docs/api/otherExports.mdx:35]:
```suggestion
[The `original` function](https://immerjs.github.io/immer/docs/original) from the [`immer` library](https://immerjs.github.io/immer/), which returns the original object. This is particularly useful for referential equality check in reducers.
```

**Comment by jackielii** [comment]
hmm, the pipeline failing, but seems not related to this PR:

```
FAIL type-tests/types.test.ts (45.576s)
  ✕ Types (43152ms)

  ● Types

    Semantic: /home/runner/work/redux-toolkit/redux-toolkit/src/index.ts (8, 3) Module '"../node_modules/immer/compat/pre-3.7/dist/immer"' has no exported member 'freeze'.
```

**Comment by phryneas** [comment]
Yes and no.
This segment should add an `// @ts-ignore` before running the tests in old TS versions.

https://github.com/reduxjs/redux-toolkit/blob/master/.github/workflows/tests.yml#L72-L76

Maybe the export changed there now?

**Comment by jackielii** [comment]
@phryneas thanks for the tip. I believe it's now passing except for the same error from master

**Comment by phryneas** [comment]
Yeah, Travis is just failing because we want to remove that soon anyways and I didn't update anything there ;)

I'll merge this into the `next` branch. Thank you!

**Pull request merged**